### PR TITLE
Remove forking logger

### PR DESCRIPTION
**Pull request by laurentiuNiculae**
- no longer needed, the race conditions were fixed

**What type of PR is this?**
cleanup

**Which issue does this PR fix**:

**What does this PR do / Why do we need it**:
We are cleaning the forks made to the logger because they are no longer needed. Issue with race condition fixed in #817 

**If an issue # is not available please add repro steps and logs showing the issue**:

**Testing done on this change**:

**Automation added to e2e**:

**Will this break upgrades or downgrades?**

**Does this PR introduce any user-facing change?**:

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by peusebiu — APPROVED** [review]
lgtm

File: pkg/extensions/extension_metrics.go [hunk 1]
```diff
@@ -26,10 +26,8 @@ func EnableMetricsExtension(config *config.Config, log log.Logger, rootDir strin
 }
 
 func SetupMetricsRoutes(config *config.Config, router *mux.Router, storeController storage.StoreController,
-	l log.Logger,
+	log log.Logger,
 ) {
-	// fork a new zerolog child to avoid data race
-	log := log.Logger{Logger: l.With().Caller().Timestamp().Logger()}
```
Comment (rchincha) [inline pkg/extensions/extension_metrics.go:32]: Just make sure the original logger has all these options set up.

Comment (laurentiuNiculae) [reply pkg/extensions/extension_metrics.go:32]: The original logger has those options. All loggers created with NewLogger are created like this:
```
Logger{Logger: log.Hook(goroutineHook{}).With().Caller().Timestamp().Logger()}
```

**Review by rchincha — APPROVED** [review]
lgtm

**Pull request merged**